draft-ietf-tls-tls13.md

```diff
@@ -3687,7 +3690,7 @@ content type of the message.
 This padding scheme was selected because it allows padding of any encrypted
 TLS record by an arbitrary size (from zero up to TLS record size
 limits) without introducing new content types.  The design also
-enforces all-zero padding octets, which allows for quick detection of
```

Comment:
I think the current text is preferable here. Both might be grammatically correct, but the former sounds better.

(design allows for; octets allow for)